src/test/regress/expected/isolation_create_restore_point.out

```diff
@@ -102,13 +102,14 @@ step s1-copy:
 
 step s2-create-restore:
 	SELECT 1 FROM citus_create_restore_point('citus-test');
+ <waiting ...>
```

Comment:
`citus_create_restore_point ` is blocked until copy  finishes.  Restore point  acquires `ExclusiveLock` on pg_dist_node, COPY now takes  `ShareLock`.

I guess this could become an issue for cloudplanes? I'll think about on this, if you  have any suggestions, please  share